sdk/python/client.py

```diff
@@ -1,51 +1,142 @@
 import grpc
 import codegen.oomd_pb2
 import codegen.oomd_pb2_grpc
+import time
+from pathlib import Path
+from subprocess import Popen
 
 # Convert google.protobuf.pyext._message.MessageMapContainer object to Python dictionary
 def map_container_to_dict(map_container):
-    return dict({k: getattr(v, v.WhichOneof('kind')) for k, v in map_container.items()})
+    return dict({k: getattr(v, v.WhichOneof("kind")) for k, v in map_container.items()})
+
 
 class Client(object):
-    def __init__(self):
-        pass
+    def __init__(self, port: int, config_path: str):
+        try:
+            self.oomd = Popen(["oomd", "--config", config_path])
+        except e:
```

Comment:
`try ... except e: raise e`

doesn't seem to do anything 🤔

Reply:
If this is the case, just remove `try .. except ...` and let it fail

Reply:
Some changes were made